scheduler/test/cook/test/rest/api.clj

```diff
@@ -1492,9 +1492,9 @@
                   (is (= (assoc (expected-job-map job)
                            :container (assoc-in docker-container
                                                 [:docker :parameters]
-                                                [{:key "user" :value "1234:2345"}
-                                                 {:key "tee" :value "tie"}
-                                                 {:key "fee" :value "fie"}]))
+                                                [{:key "tee" :value "tie"}
+                                                 {:key "fee" :value "fie"}
+                                                 {:key "user" :value "1234:2345"}]))
```

Comment:
The changes to this test are unrelated to the code changes in this PR. The test was failing, though, so I fixed it to get the build green.

Reply:
just change to make it a set and update-in set on api/fetch-job-map

Reply:
Changed the test to use `frequencies`